source/common/runtime/runtime_features.cc

```diff
@@ -131,5 +135,25 @@ RuntimeFeatures::RuntimeFeatures() {
   }
 }
 
+bool RuntimeFeatures::enabledByDefault(absl::string_view feature) const {
+  if (feature.substr(0, ConstantQuicFlagPrefix.length()) == ConstantQuicFlagPrefix) {
+    auto res = quiche::FlagRegistry::getInstance().findFlag(absl::StrCat(
+        "FLAGS_quic_reloadable_flag_", feature.substr(ConstantQuicFlagPrefix.length() - 4)));
+    ASSERT(res != nullptr);
```

Comment:
Would this crash if someone sets a runtime flag "envoy.reloadable_features.quic.foo" to false?
I'd say if it doesn't exist we should fail over to normal runtime semantics, or just log a warning and continue.

Reply:
Here we only assert that any GetQuicReloadableFlag() call in our code actually calls on an existing flag.
If someone mistakenly sets a runtime flag override, it won't have any effect because there will be no call for it.